packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -238,6 +238,9 @@ - (void)appLoaderTask:(EXUpdatesAppLoaderTask *)appLoaderTask didStartLoadingUpd
 
 - (void)appLoaderTask:(EXUpdatesAppLoaderTask *)appLoaderTask didFinishWithLauncher:(id<EXUpdatesAppLauncher>)launcher isUpToDate:(BOOL)isUpToDate
 {
+  if (_remoteLoadStatus == EXUpdatesRemoteLoadStatusLoading && isUpToDate) {
```

Comment:
If isUpToDate is false, do we still want the remote load status to be "loading"?

Reply:
Yes, that's an indication that an update is still downloading in the background (this method was called because the timeout was reached and so a cached update is being launched). I'll add a comment clarifying this.